handlers/upload.go

```diff
@@ -112,6 +112,8 @@ func (d *CatalystAPIHandlersCollection) UploadVOD() httprouter.Handle {
 }
 
 func (d *CatalystAPIHandlersCollection) processUploadVOD(streamName, sourceURL, targetURL string) error {
+	// TODO: remove this logic to force input URLs to mp4/mov format
+	sourceURL = "mp4:" + sourceURL
```

Comment:
Will mist handle mp4:https:// fine?

Reply:
Yep it discards mp4: and uses it to know that the source URL is an mp4/mov file